Mathlib/LinearAlgebra/ExteriorPower/Basic.lean

```diff
@@ -0,0 +1,427 @@
+  unfold liftAlternating_aux
+  simp only [eq_mpr_eq_cast, LinearMap.pi_apply, cast_eq, dite_eq_ite, ite_true,
+    LinearMap.id_coe, id_eq]
+
+@[simp] lemma liftAlternating_comp_ιMulti (f : AlternatingMap R M N (Fin n)) :
+    (LinearMap.compAlternatingMap (liftAlternating n f)) (ιMulti R n) = f := by
+  ext a
+  simp only [LinearMap.compAlternatingMap_apply]
+  rw [liftAlternating_apply_ιMulti]
+
+@[simp] lemma liftAlternating_ιMulti :
+    liftAlternating n (R := R) (M := M) (ιMulti R n) = LinearMap.id := by
+  ext u
+  simp only [liftAlternating_comp_ιMulti, ιMulti_apply, LinearMap.compAlternatingMap_apply,
+    LinearMap.id_coe, id_eq]
+
+/-- If `f` is an alternating map from `M` to `N`, `liftAlternating n f` is the corresponding
+linear map from `Λ[R]^n M` to `N` and `g` is a linear map from `N` to `N'`, then
+the alternating map `g.compAlternatingMap f` from `M` to `N'` corresponds to the linear
+map `g.comp (liftAlternating n f)` on `Λ[R]^n M`. -/
+@[simp]
+lemma liftAlternating_comp (g : N →ₗ[R] N') (f : AlternatingMap R M N (Fin n)) :
+    liftAlternating n (g.compAlternatingMap f) = g.comp (liftAlternating n f) := by
+  ext u
+  simp only [liftAlternating_comp_ιMulti, LinearMap.compAlternatingMap_apply, LinearMap.coe_comp,
+    Function.comp_apply, liftAlternating_apply_ιMulti]
+
+/-- The linear equivalence between `n`-fold alternating maps from `M` to `N` and linear maps from
+`Λ[R]^n M` to `N`. -/
+@[simps!]
+def liftAlternatingEquiv : AlternatingMap R M N (Fin n) ≃ₗ[R] (Λ[R]^n) M →ₗ[R] N :=
+  LinearEquiv.ofLinear (liftAlternating n)
+  {toFun := fun F ↦ F.compAlternatingMap (ιMulti R n)
+   map_add' := by intro F G; ext x
+                  simp only [LinearMap.compAlternatingMap_apply, LinearMap.add_apply,
+                    AlternatingMap.add_apply]
+   map_smul' := by intro a F; ext x
+                   simp only [LinearMap.compAlternatingMap_apply, LinearMap.smul_apply,
+                     RingHom.id_apply, AlternatingMap.smul_apply]}
+  (by ext _; simp only [LinearMap.coe_comp, LinearMap.coe_mk, AddHom.coe_mk, Function.comp_apply,
+        liftAlternating_comp, liftAlternating_ιMulti, LinearMap.comp_id,
+        LinearMap.compAlternatingMap_apply, LinearMap.id_coe, id_eq])
+  (by ext _; simp only [LinearMap.coe_comp, LinearMap.coe_mk, AddHom.coe_mk, Function.comp_apply,
+        liftAlternating_comp_ιMulti, LinearMap.id_coe, id_eq])
+
+lemma liftAlternatingEquiv_apply (f :AlternatingMap R M N (Fin n)) (x : (Λ[R]^n) M) :
+    ExteriorPower.liftAlternatingEquiv R n f x = ExteriorPower.liftAlternating n f x := rfl
+
+@[simp]
+lemma liftAlternatingEquiv_symm_apply (F : (Λ[R]^n) M →ₗ[R] N) (m : Fin n → M) :
+    (ExteriorPower.liftAlternatingEquiv R n).symm F m = F.compAlternatingMap (ιMulti R n) m := rfl
+
+/-! Functoriality of the exterior powers. -/
+
+variable {R}
+
+/-- The linear map between `n`th exterior powers induced by a linear map between the modules. -/
+def map (f : M →ₗ[R] N) : (Λ[R]^n) M →ₗ[R] (Λ[R]^n) N := by
```

Comment:
Doesn't this follow directly as an application of `liftAlternating`?

Reply:
Yes, this is `liftAlternating _ <| AlternatingMap.compLinearMap (ιMulti _ _) f`

Reply:
I went ahead and pushed this in 9a527a7d50bb778bae21f62425070c569ea031e6, hope that's ok; feel free to revert if not.

Reply:
Yes, of course that's okay, thanks !